collectors/mesos/agent/agent.go

```diff
@@ -36,25 +36,20 @@ const (
 	HTTPTIMEOUT = 2 * time.Second
 )
 
-var (
-	maColLog = log.WithFields(log.Fields{
-		"collector": "mesos-agent"})
-)
-
 // Collector defines the collector type for Mesos agent. It is
 // configured from main from config file options and pass a new instance of HTTP
 // client and a channel for dropping metrics onto.
 type Collector struct {
 	Port            int           `yaml:"port"`
 	PollPeriod      time.Duration `yaml:"poll_period"`
 	RequestProtocol string        `yaml:"request_protocol"`
-
-	MetricsChan chan producers.MetricsMessage
-	NodeInfo    collectors.NodeInfo
-	HTTPClient  *http.Client
+	HTTPClient      *http.Client
 
 	agentState       agentState
 	containerMetrics []agentContainer
+	log              *logrus.Entry
```

Comment:
Nit: For people who may be unfamiliar, if might be nice to drop in a comment to documentation `https://godoc.org/github.com/sirupsen/logrus#Entry`